### PR TITLE
zh-trans: example-task-template.md

### PR DESCRIPTION
**Pull request by wzqnls**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tasks/example-task-template.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: Example Task Template
```
Comment (Mr-Linus) [inline content/zh/docs/tasks/example-task-template.md]: 标题可以翻译下

File: content/zh/docs/tasks/example-task-template.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+{{% capture whatsnext %}}
+
+<!-- **[Optional Section]** -->
+
+**[可选部分]**
+
+<!-- * Learn more about [Writing a New Topic](/docs/home/contribute/write-new-topic/).
+* See [Using Page Templates - Task template](/docs/home/contribute/page-templates/#task_template) for how to use this template. -->
+
+* 学习更多关于 [写一个新主题](/docs/home/contribute/write-new-topic/)。
```
Comment (Mr-Linus) [inline content/zh/docs/tasks/example-task-template.md]:
```suggestion
* 学习更多关于[写一个新主题](/docs/home/contribute/write-new-topic/)。
```

File: content/zh/docs/tasks/example-task-template.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+* 阅读 [使用页面模板 - 任务模板](/docs/home/contribute/page-templates/#task_template) 学习如何使用这些模板。
```
Comment (Mr-Linus) [inline content/zh/docs/tasks/example-task-template.md]:
```suggestion
* 阅读[使用页面模板 - 任务模板](/docs/home/contribute/page-templates/#task_template)学习如何使用这些模板。
```

File: content/zh/docs/tasks/example-task-template.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+<!-- ## Doing ... -->
+
+## 做... 
+
+<!-- 1. Do this.
+1. Do this next. Possibly read this [related explanation](...). -->
+
+1. 做这个。
+1. 下一步做这个. 可以阅读 [相关解释](...).
```
Comment (Mr-Linus) [inline content/zh/docs/tasks/example-task-template.md]:
```suggestion
1. 下一步做这个. 可以阅读 [相关解释](...)。
```

File: content/zh/docs/tasks/example-task-template.md (hunk 1 above)
Comment (wzqnls) [reply content/zh/docs/tasks/example-task-template.md]: sorry  vscode忘记保存了

**Comment by markthink** [comment]
建议 title 标题译文放到最前面, 避免渲染有问题。

**Pull request merged**